doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -172,8 +173,10 @@ Doc. ID: UG-CS
   - [17.3 Correlation ID HTTP header](#173-correlation-id-http-header)
   - [17.4 Data Integrity errors](#174-data-integrity-errors)
   - [17.5 Warning responses](#175-warning-responses)
-- [18. Migrating to Remote Database Host](#18-migrating-to-remote-database-host)
-  - [19 Additional Security Hardening](#19-additional-security-hardening)
+- [18. Remote Database Host](#18-remote-database-host)
+  - [18.1 Migrating to Remote Database Host](#181-migrating-to-remote-database-host)
+  - [18.2 Passing additional parameters to psql](#182-passing-additional-parameters-to-psql)
```

Comment:
I think we should have this as a completely separate step, since you could technically do this for local databases as well, even if that use case is probably pretty rare.